open_instruct/ppo_vllm_thread_ray_gtrl.py

```diff
@@ -1510,7 +1511,10 @@ def launch_ai2_evals_on_weka(self, step_dir: str, training_step: Optional[int] =
     --run_safety_evaluations \
     --run_id {wandb_url} \
     --step {training_step} \
+    --oe_eval_max_length {args.eval_max_length} \
     --skip_oi_evals"""
+            if training_step is not None:
+                command += f" --step {training_step}"
```

Comment:
This is duplicated? 

<img width="767" alt="image" src="https://github.com/user-attachments/assets/4f62d887-9f26-45dd-a0a8-0d0446a96f43" />

Reply:
Ah!